sunpy/map/mapbase.py

```diff
@@ -198,6 +198,11 @@ def __init__(self, data, header, plot_settings=None, **kwargs):
 
         # Setup some attributes
         self._nickname = None
+        self._default_time = None
```

Comment:
A comment above these describing the purpose of them (to reduce repeat warnings) would be nice.